appveyor.yml

```diff
@@ -2,8 +2,6 @@ version: '{build}'
 os: Visual Studio 2015
 install:
 - set PATH=%ProgramFiles(x86)%\MSBuild\14.0\Bin;%PATH%
-- dnvm update-self
-- dnvm install 1.0.0-rc1-update1
```

Comment:
Should this run the equivalent `dotnet` command to ensure RC2 is installed on the build machine?